framework/include/systems/NonlinearEigenSystem.h

```diff
@@ -125,7 +138,7 @@ class NonlinearEigenSystem : public NonlinearSystemBase
    *
    * @return all converged eigenvalues as complex numbers
    */
-  virtual const std::vector<std::pair<Real, Real>> & getAllConvergedEigenvalues()
+  virtual const std::vector<std::pair<Real, Real>> & getAllConvergedEigenvalues() const
```

Comment:
Oh I love `const`. Are you actually going to override this method somewhere? Wondering if this needs to be `virtual`. I always thought that `NonlinearEigenSystem` was a leaf object, but maybe you derive from it in Griffin or something.